eval/utils.py

```diff
@@ -291,6 +291,29 @@ def load_hf_lm_and_tokenizer(
         
     return model, tokenizer
 
+def load_hf_tokenizer(
```

Comment:
Since we now have the tokenizer loading function separate, I think it's better to change the other function to only load the model.